base/interactiveutil.jl

```diff
@@ -251,7 +251,8 @@ function versioninfo(io::IO=STDOUT, verbose::Bool=false)
         println(io, "DEBUG build")
     end
     println(io,             "Platform Info:")
-    println(io,             "  System: ", Sys.KERNEL, " (", Sys.MACHINE, ")")
+    println(io,             "  OS: ", is_windows() ? "Windows" : is_apple() ?
```

Comment:
On Windows, it also looks like it prints out a kernel version (e.g. I get "Version 6.2.9200" on Windows 8) rather than the familiar OS version number.

On Debian, `versioninfo(true)` does print out some distro info.

In any case, I'd rather have this info in `versioninfo(false)` as well.